venonactl/pkg/store/store.go

```diff
@@ -99,6 +139,36 @@ func GetStore() *Values {
 }
 
 func (s *Values) BuildValues() map[string]interface{} {
+	if s.Runner == nil {
```

Comment:
not sure why we are using pointers if we are always initializing them. I thought we did so we can check in the gotemplate if they are nil. but now they are always initialized, so we need to let go do this for us by using zero values.